sklearn/isotonic.py

```diff
@@ -153,13 +153,14 @@ class IsotonicRegression(RegressorMixin, TransformerMixin, BaseEstimator):
         or decrease with `X`. 'auto' will decide based on the Spearman
         correlation estimate's sign.
 
-    out_of_bounds : str, default="nan"
+    out_of_bounds : {'nan', 'clip', 'raise'}, default='nan'
         The ``out_of_bounds`` parameter handles how `X` values outside of the
-        training domain are handled.  When set to "nan", predictions
-        will be NaN.  When set to "clip", predictions will be
-        set to the value corresponding to the nearest train interval endpoint.
-        When set to "raise" a `ValueError` is raised.
+        training domain are handled.
 
+        - `'nan'`, predictions will be NaN.
```

Comment:
we usually don't wrap strings in backquotes since the `''` delimiters are enough (or at least I don't).